doc/user/COMMANDS.md

```diff
@@ -0,0 +1,78 @@
+# Commands
+
+F3D will provides access to commands through interactions configuration and command line widget in the future.
+Commands let you trigger specific behavior that may not be available otherwise.
+Please note commands are currently experimental and the behavior, names and arguments may change without deprecation.
+
+Standard commands have the following syntax:
+
+`action libf3d.option.name [values]`
+
+Specific commands have the following syntax:
+
+`specific_action [arg] [values]`
+
+See below for the details.
+
+## libf3d provided commands
+
+The libf3d provided a few commands, many related to manipulating libf3d (option)[../libf3d/OPTIONS.md].
+
+### Standard commands
+
+`set option.name values`: A command to set a libf3d option, eg: `set scene.up.direction +Z`.
+
+`toggle option.name`: A command to toggle a boolean libf3d option, eg: `toggle ui.scalar_bar`. 
+
+`unset option.name`: A command to unset an optional libf3d option, eg: `unset render.hdri.file`. 
+
+`reset option.name`: A command to reset a libf3d option to its default values, eg: `reset render.background.blur_coc`.
```

Comment:
What's the difference between `reset` and `unset`? If I'm using `reset` on an optional option, I'm expecting it to be unset, so it feels like it's redundant to me.

Reply:
I agree that technically reset and unset is similar, so I could keep only reset.

Reply:
Yes I think `reset` only should be kept